stdlib/2and3/_types.pyi

```diff
@@ -0,0 +1,7 @@
+from typing import Protocol, Union
```

Comment:
This should have a comment explaining that it's for typeshed-only objects that don't exist at runtime.